pkg/config/blockfetcher_config.go

```diff
@@ -44,8 +44,5 @@ func (cfg *NeoFSBlockFetcher) Validate() error {
 	if len(cfg.Addresses) == 0 {
 		return errors.New("addresses are not set")
 	}
-	if !cfg.SkipIndexFilesSearch && cfg.IndexFileSize == 0 {
-		return errors.New("IndexFileSize is not set")
-	}
```

Comment:
I can't leave comment at the proper place but:

> if cfg.BQueueSize < cfg.OIDBatchSize {

This check must be performed only if both values are set. If they are not set, then default value should be used and hence, this check must be skipped.

Reply:
And BTW, this is a separate change. Don't mix two commits. One commit is responsible for moving constants to a separate package, and another commit is responsible for defaults tuning.